exercises/Functions-1-R.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: exercise
+title: Functions 1
+language: R
+---
+
+Use the built-in functions `abs()`, `round()`, `sqrt()`, `as.integer()`, `as.numeric()`, and `as.character()` to print out the answers to the following. A built-in function is one
+that you don't need to import a module to use. Use another function, `help()`,
```

Comment:
Done.

'Load a package' is directly from the R documentation. Chose to include
'install' also to be clear that is the first step.

On Wed, Apr 15, 2015 at 4:28 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Functions-1-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/13#discussion_r28464930>
> :
>
> > @@ -0,0 +1,23 @@
> > +---
> > +layout: exercise
> > +title: Functions 1
> > +language: R
> > +---
> > +
> > +Use the built-in functions `abs()`, `round()`, `sqrt()`, `as.integer()`, `as.numeric()`, and `as.character()` to print out the answers to the following. A built-in function is one
> > +that you don't need to import a module to use. Use another function, `help()`,
>
> "import a module" -> "load a package" or something else more R'y
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/13/files#r28464930>
> .
>